File: eBUS_SDK/eBUS_3_1_9_3133/SamplesDotNet/PvDualSourceSample/BrowserForm.cs
// *****************************************************************************
//
//     Copyright (c) 2012, Pleora Technologies Inc., All rights reserved.
//
// *****************************************************************************

using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace PvDualSourceSample
{
    public partial class BrowserForm : Form
    {
        public BrowserForm()
        {
            InitializeComponent();
        }

        private void BrowserForm_FormClosing(object sender, FormClosingEventArgs e)
        {
            if (!Modal && (e.CloseReason == CloseReason.UserClosing))
            {
                e.Cancel = true;
                Hide();
            }
        }
    }
}

File: eBUS_SDK/eBUS_5_1_5_4563/SamplesDotNet/SimpleApplicationVB/BrowserForm.vb
' ****************************************************************************
'
'     Copyright (c) 2012, Pleora Technologies Inc., All rights reserved.
'
' ****************************************************************************

Public Class BrowserForm
    Private Sub BrowserForm_FormClosing(ByVal sender As System.Object, ByVal e As System.Windows.Forms.FormClosingEventArgs) Handles MyBase.FormClosing
        If Modal <> True And e.CloseReason = CloseReason.UserClosing Then
            e.Cancel = True
            Hide()
        End If
    End Sub
End Class



